src/lib/ext-activitylog.js

```diff
@@ -95,6 +99,14 @@ class Model {
     return true;
   }
 
+  matchFilterTabId({ tabId }) {
+    if (this.filter.tabId === null) {
```

Comment:
`== null` to make sure this return earlier if tabId is `null` or `undefined`.

You may want to try this in a js console (e.g. a devtools webconsole panel or from a nodejs interactive interpreter) to better see the difference, eg. when using  `... == null`:

```
null == null // => true
undefined == null // => true
0 == null // false
```

versus when using `... === null`:

```
null === null // => true
undefined === null // => false
0 === null // => false
```

(but leave the `===` at line 107, there we want it to only return true on strict equality between the two values).

Reply:
Thanks :-).  Updated with double equals :-)